data/json/items/generic/bathroom_house.json

```diff
@@ -115,6 +115,9 @@
     "name": { "str": "comb" },
     "type": "GENERIC",
     "description": "A grooming tool with teeth for straightening your hair.",
+    "weight": "9 g",
+    "volume": "80 ml",
```

Comment:
You are actually right on this. I believe I fucked up here and I believe I calculated the volume of the packaging instead of the item volume. Almost made the same mistake with plastic utensils. I'll review that sometime that I'm back at my computer.